zuul.d/jobs.yaml

```
@@ -458,7 +507,7 @@
     name: openstacksdk-ansible-functional-devstack
     parent: init-test
     description: |
-      Run openstacksdk ansible functional tests against a master devstack using devel branch version of ansible.
+      Run openstacksdk ansible functional tests against a master devstack using devel branch of ansible.
```

```suggestion
      Run openstacksdk ansible functional tests against a master OpenStack using devel branch of ansible.
```

there are same issue exist in the following description

---

done